pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraItemsOnOrder.scala

```diff
@@ -59,9 +60,12 @@ object SierraItemsOnOrder extends Logging {
     (getStatus(order), getOrderDate(order), getReceivedDate(order)) match {
 
       // status 'o' = "On order"
+      // status 'c' = "Serial on order"
```

Comment:
What is status `a`?

Reply:
status 'a' = "fully paid". It's mentioned in the comment on the next branch, but let me add it here.

```suggestion
      // status 'c' = "Serial on order"
      // status 'a' = "fully paid"
```